auto_circuit/utils/patch_wrapper.py

```diff
@@ -85,11 +85,29 @@ def __init__(
             self.mask_fn: MaskFn = None
             self.dropout_layer: t.nn.Module = t.nn.Dropout(p=0.0)
         self.patch_mode = False
+        self.batch_size = None
 
         assert head_dim is None or seq_dim is None or head_dim > seq_dim
         dims = range(1, max(head_dim if head_dim else 2, seq_dim if seq_dim else 2))
         self.dims = " ".join(["seq" if i == seq_dim else f"d{i}" for i in dims])
 
+    def set_mask_batch_size(self, batch_size: int | None):
```

Comment:
Please add docstring. With warning that this should not be used directly, but with context manager.

You can insert a link to the context manager in the docstring using this notation:
[`set_mask_batch_size`][auto_circuit.utils.graph_utils.set_mask_batch_size]

Reply:
And probably also include the warnings about this breaking various features of the library and to be used with caution etc.